### ISSUE TITLE
USER ISSUE: After some time cant connect to a server

### ISSUE THREAD

**Issue by organicfarmer** [StrangeLoopGames/EcoIssues#9433]
**Version:** 0.7.7.2 beta

**Steps to Reproduce:**
play a few days on server, then cant connect any more. Just says connection timed out. Dont get a ping value for that server anymore although it is running. Has happened twice. However - after a few days i can now log on to the the first server it happened on again - GoW 

**Expected behavior:**

**Actual behavior:**
connection failed connection timed outn
**Do you have mods installed? Does issue happen when no mods are installed?:**
no mods

This is the "message that goes into the clipboard from the game:

<size=150%>Connection Failed</size>

Failed to connect - connection timed out.

Log File:
Initialize engine version: 2018.2.0f2 (787658998520)
GfxDevice: creating device client; threaded=1
Direct3D:
    Version:  Direct3D 11.0 [level 11.1]
    Renderer: NVIDIA GeForce GTX 1080 (ID=0x1b80)
    Vendor:   
    VRAM:     8079 MB
    Driver:   23.21.13.9077
Begin MonoManager ReloadAssembly
- Completed reload, in  0.114 seconds
<RI> Initializing input.

<RI> Input initialized.

<RI> Initialized touch support.

UnloadTime: 0.319445 ms
Client version: 0.7.7.2 beta
Tuesday, September 11, 2018 12:08 PM
 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug.bindings.h Line: 43)

Disconnected from server.
 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug.bindings.h Line: 43)

Disconnected from server.
 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug.bindings.h Line: 43)

**Comment by BlazeGuy2** [comment]
This will be tough to address without a reliable way to reproduce this issue.  What's the server's ip address?

**Comment by Leadenigma** [comment]
199.116.115.66:3000

**Comment by organicfarmer** [comment]
I can add that I have installed/reinstalled the game; reset the modem/router. I have the very same issue with the test-server. It just shows up with no ping value.
![image](https://user-images.githubusercontent.com/43418072/45776378-f810f080-bc52-11e8-8844-58849481c16e.png)

**Comment by organicfarmer** [comment]
This is what I get when browsing all servers - a lot of "no-pingers" at the bottom
![image](https://user-images.githubusercontent.com/43418072/45776466-40c8a980-bc53-11e8-8e91-c41f450b8af1.png)

**Comment by D3nnis3n** [comment]
In the case of the GoW server this happened because our firewall blocked some people playing as it thought they would be attackers - as of any reason i don't really know yet. We reconfigured it to be not that harsh and everything works now. I saw organicfarmer being online today fine and several other peoples that had the issues. I personally would suggest this issue is with the servers - not with the clients.
